### PR TITLE
Adds Ledger Documentation

### PR DESCRIPTION
**Pull request by cordt-sei**
Adds `Ledger Setup` to `User Guides` as per [Ledger Device App Submission](https://developers.ledger.com/docs/device-app/deliver/submission-form) requirements.

Builds and runs locally without issue. [Preview link](http://basementnodes.ca:3000/user-guides/ledger-setup)

### CODE REVIEW

File: pages/user-guides/ledger-setup.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+The Sei network supports various crypto assets, including:
+
+- Sei tokens (SEI, custom tokenfactory tokens)
+- CW20 tokens
+- CW721 NFTs
+- ERC20 tokens
+- ERC721 NFTs
+
+Using a Ledger device with the Sei network ensures secure key management while interacting with the blockchain.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
Using a Ledger hardware wallet ensures secure key management while interacting with the Sei network, which supports various crypto assets, including:
```

Comment (ghost) [reply pages/user-guides/ledger-setup.mdx]: And then I think we can replace the first statement in this section

File: pages/user-guides/ledger-setup.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+For further assistance, you can reach out to the following support channels:
+
+- **Sei Network Community**: Join the official Sei network community [Telegram](https://t.me/seinetwork) or [Discord](https://discord.gg/sei).
+- **Ledger Support**: For Ledger-specific issues, visit the [Ledger Support Center](https://support.ledger.com/hc/en-us).
+- **Wallet Support**: For wallet-related questions, check the support resources provided by your chosen wallet application.
+
+## Additional Resources
+
+- **Sei Network Documentation**: Refer to the official Sei network documentation for detailed information about the network and its features.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
- **Sei Network Documentation**: Detailed information about the Sei network is available in the [official documentation](https://www.docs.sei.io/).
```

File: pages/user-guides/ledger-setup.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- [Ledger Academy](https://www.ledger.com/academy): Learn more about cryptocurrency and security.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
- **Ledger Academy**: Learn more about cryptocurrency security on the official [Ledger Academy](https://www.ledger.com/academy).
```

File: pages/user-guides/ledger-setup.mdx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+</Callout>
+
+3. **Share the Address**:
+   - Copy the verified address and share it with the sender or paste it into an exchange withdrawal form.
+
+## How to Send Crypto Assets
+
+1. **Navigate to the Send Section**:
+
+   - Go to the **Send** or similar section in your wallet application.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
   - Navigate to the **Send** section (or equivalent) in your wallet application.
```

File: pages/user-guides/ledger-setup.mdx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+3. Your account balance and asset details should be displayed on the main screen of the wallet application.
+
+## How to Receive Crypto Assets
+
+1. **Navigate to the Receive Section**:
+
+   - Go to the **Receive** or similar section in your wallet application.
+   - Select the crypto asset (if applicable) you wish to receive.
+
+2. **Verify the Receive Address**:
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
2. **Verify the Receiver Address**:

```

File: pages/user-guides/ledger-setup.mdx [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+   - The wallet application will display the receive address.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
   - The wallet application will display the receiver address.
```

File: pages/user-guides/ledger-setup.mdx [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+   - Verify that the address shown on the screen matches the one displayed on your Ledger device.
+   - Confirm the address on your Ledger device by pressing the appropriate buttons.
+
+<Callout type='warning'>
+	Always verify the receive address on your Ledger device before sharing it. This helps prevent potential attacks and ensures you're using the correct address.
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx]:
```suggestion
	Always verify the receiver address on your Ledger device before sharing it. This prevents attacks like address spoofing.
```

File: pages/user-guides/ledger-setup.mdx [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+### Using a Compatible Wallet Application
+
+1. **Connect Your Ledger Device**:
+
+   - Open your compatible Sei network wallet application.
+   - Connect your Ledger device to your computer.
+   - Unlock your Ledger device and open the Sei app.
+
+2. **Access the Sei Network**:
```
Comment (ghost) [inline pages/user-guides/ledger-setup.mdx:61]:
```suggestion
2. **Access the Sei Network**:

```

**Review by ghost — APPROVED** [review]

File: pages/user-guides/ledger-setup.mdx (hunk 8 above)
Comment (cordt-sei) [reply pages/user-guides/ledger-setup.mdx:61]: Not sure what changes here, but the rest were good so I trust it.

**Review by besated — APPROVED** [review]

**Pull request merged**